palvelutarjotin/settings.py

```diff
@@ -294,6 +295,8 @@
     "DATA_SOURCE": env.str("LINKED_EVENTS_DATA_SOURCE"),
 }
 
+SERVICEMAP_API_CONFIG = {"ROOT": env.str("SERVICEMAP_API_ROOT")}
```

Comment:
Just FYI, since you've defined `SERVICEMAP_API_ROOT` as a string, you can here also just use `env("SERVICEMAP_API_ROOT")`